packages/devtools_app/lib/src/shared/device_dialog.dart

```diff
@@ -157,7 +156,7 @@ class _VMFlagsDialogState extends State<VMFlagsDialog> with AutoDisposeMixin {
   }
 
   void _updateFromController() {
-    flags = infoController.flagListNotifier.value.flags
+    flags = infoController.flagListNotifier.value!.flags!
```

Comment:
we should either make sure flagListNotifier.value cannot return null (maybe empty list instead), or handle a possible null value here e.g. `infoController.flagListNotifier.value?.flags ?? []`

Reply:
done